packages/decentraland-ecs/src/decentraland/Types.ts

```diff
@@ -51,6 +52,10 @@ export type DecentralandInterface = {
   /** set a new parent for the entity */
   setParent(entityId: string, parentId: string): void
 
+  // QUERY
+
+  query(system: string, config: any): void
```

Comment:
The `system`/`config` naming seems iffy to me as we use the word `system` for the ECS systems and its not consistent with the rest of the messages. 
```suggestion
  query(queryId: string, payload: any): void
```